### PR TITLE
Update user-panel page copy update

### PR DESCRIPTION
**Pull request by davidpmccormick**
☝️ [as per comment on #5271](https://github.com/wellcomecollection/wellcomecollection.org/issues/5271#issuecomment-653417085)

### CODE REVIEW

File: content/webapp/pages/user-panel.js [hunk 1]
```diff
@@ -86,29 +86,21 @@ const UserPanelPage = () => {
...
+                    questions like these as part of our user panel.
                   </p>
+
                   <h2>Who can take part?</h2>
                   <p>
-                    {`We're looking for people who visit our exhibitions, read our
-                    stories, attend our events and use our library and
-                    collections - anyone who's interested in helping improve
-                    Wellcome Collection's services for everyone.`}
+                    {`We’re looking for people who visit our exhibitions, read our stories, attend our events and use our Library and collections – anyone who’s interested in helping us improve our services.`}
```
Comment (jamesgorrie) [inline content/webapp/pages/user-panel.js:97]: We're using `’` and `'` differently, which should it be?

Comment (davidpmccormick) [reply content/webapp/pages/user-panel.js:97]: good spot – I _think_ we tend to favour smart quotes – I'll check and update accordingly.

Comment (davidpmccormick) [reply content/webapp/pages/user-panel.js:97]: I've not yet found a page where there has been consistent use of either `’` or `'`. Which suggests that this is something that needs broader consideration. But I've updated these to be all `’`.

Comment (davidpmccormick) [reply content/webapp/pages/user-panel.js]: Opened #5348 so we can deal with this

**Pull request merged**